google_servers.py

```diff
@@ -0,0 +1,15 @@
+# Dependency injection
+class GoogleServers:
+    pass
+
+class GoogleCalendar(GoogleServers):
```

Comment:
You can’t inject dependency with inheritance. You should’ve used aggregation here